### PR TITLE
Fix partner packs metadata 

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28213

## Description
Changed Irondefense pack author to Iron defense 
Changed author snd support for DomainTools Iris (moved test out of test suit as the partner is non-certified) 
Changed illuminate pack author to Analyst Platform

### CODE REVIEW

**Review by Itay4 — APPROVED** [review]

File: Tests/conf.json [hunk 1]
```diff
@@ -2263,11 +2263,6 @@
             "integrations": "Exabeam",
             "playbookID": "Exabeam - Test"
         },
-        {
-            "integrations": "DomainTools Iris",
```
Comment (yaakovi) [inline Tests/conf.json:2267]: can we also remove the integration from `skip`?

**Review by yaakovi — APPROVED** [review]
wait with merging this PR until we talk about the test playbook move

File: Tests/conf.json (hunk 1 above)
Comment (ShahafBenYakir) [reply Tests/conf.json:2267]: @yaakovi I don't see that the integration is skipped... where are you looking?

**Comment by ShahafBenYakir** [comment]
Guardrails have an internal error - can't see any alert.
Validations failed on the docker image for a specific pack which is a partner pack - we can't change the docker image without the contributor approval.

**Pull request merged**